### ISSUE TITLE
How does DAO work in your project?

### ISSUE THREAD

**Issue by linonetwo** [SnowCrashDAO/metavoxel-daos#9]
Readme didn't say anything about this. So is DAO useful in your project?

**Comment by Lugdunumn** [comment]
> Readme didn't say anything about this. So is DAO useful in your project?

Your issue got deleted by project owner. Fishy and shady move.

**Comment by yy190717** [comment]
So, are you on the project side?

**Comment by luckyinghai** [comment]
> > Readme didn't say anything about this. So is DAO useful in your project?
> 
> Your issue got deleted by project owner. Fishy and shady move.

So, are you on the project side?

**Comment by linonetwo** [comment]
No, I'm also waiting for someone to answer. I assume this project is dead, and I don't know why they even try to delete issue.

**Comment by luckyinghai** [comment]
No one can reach the project team

**Comment by linonetwo** [comment]
These people are basically liers, never wanted to do project, only want money. Don't buy their tokens.
